### PR TITLE
DDCE-1746 PBIK Improvement url update and test fix

### PR DESCRIPTION
**Pull request by kuldeepwadhwa9**
# DDCE-????

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist PR Raiser
- [ ]  I've updated the README if there are functionality changes
- [ ]  I've executed the project script to check Dependencies and update them if they're non-breaking changes
- [ ]  I've executed Scalastyle and fixed any warnings that can be resolved quickly
- [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
- [ ]  I've included appropriate tests with any code I've added (Unit, Integration), and it meets Coverage
- [ ]  I've squashed my commits - including the JIRA issue number in the commit message
- [ ]  I've contacted a BA to create a task if any of these actions would result in a larger piece of work

### CODE REVIEW

File: test/controllers/ManageRegistrationControllerSpec.scala [hunk 1]
```diff
@@ -341,7 +341,7 @@ class ManageRegistrationControllerSpec extends PlaySpec with TestAuthUser with F
       val result = registrationController.updateCurrentYearRegisteredBenefitTypes.apply(mockRequestForm)
       (scala.concurrent.ExecutionContext.Implicits.global)
       status(result) must be(SEE_OTHER)
-      redirectLocation(result) must be(Some("/payrollbik/cy/add-benefit-expense-confirmed"))
+      redirectLocation(result) must be(Some("/payrollbik/cy/check-the-benefits-confirmed"))
```
Comment (anburoach) [inline test/controllers/ManageRegistrationControllerSpec.scala:344]: plz revert the change

File: conf/app.routes [hunk 2]
```diff
@@ -21,16 +21,16 @@ GET         /payrolled-benefits-expenses                         controllers.Hom
...
 POST        /cy/choose-benefit-expense                           controllers.registration.ManageRegistrationController.checkYourAnswersAddCurrentTaxYear
 
-GET         /cy/add-benefit-expense                              controllers.registration.ManageRegistrationController.showCheckYourAnswersAddCurrentTaxYear
-POST        /cy/add-benefit-expense                              controllers.registration.ManageRegistrationController.updateCurrentYearRegisteredBenefitTypes
+GET         /cy/check-the-benefits                              controllers.registration.ManageRegistrationController.showCheckYourAnswersAddCurrentTaxYear
+POST        /cy/check-the-benefits                              controllers.registration.ManageRegistrationController.updateCurrentYearRegisteredBenefitTypes
 
-GET         /:year/add-benefit-expense-confirmed                 controllers.WhatNextPageController.showWhatNextRegisteredBik(year: String)
+GET         /:year/check-the-benefits-confirmed                 controllers.WhatNextPageController.showWhatNextRegisteredBik(year: String)
```
[the remark this answers is not in the document]
Comment (kuldeepwadhwa9) [reply conf/app.routes]: Done

File: test/controllers/ManageRegistrationControllerSpec.scala (hunk 1 above)
Comment (kuldeepwadhwa9) [reply test/controllers/ManageRegistrationControllerSpec.scala]: Done
